docs/developers/guard-accounts.md

```diff
@@ -0,0 +1,134 @@
+---
+id: guard-accounts
+title: Guard accounts
+---
+
+[comment]: # (mx-abstract)
+
+On this page, you will find comprehensive information on all aspects of guarded (co-signed) transactions.
+
+[comment]: # (mx-context-auto)
+
+# Introduction
+
+Phishing attacks that scam people have become a constant problem on blockchains in general, especially when newbies to the crypto world are granted easy access to this new economy. In light of these new challenges, it is important to continue educating people on specific ways to protect themselves while also considering new methods to add protection against such attacks. That's how we came to a method that we called Guardians, with the scope of having a guardian for each account.
```

Comment:
`with the scope of having a guardian for each account.`
=>
`. This is an optional security feature that any user can enable.`